src/psyclone/dynamo0p3.py

```diff
@@ -2793,11 +2878,11 @@ def initialise(self, parent):
                 parent.add(CommentGen(parent, ""))
                 parent.add(CommentGen(parent,
                                       " Initialise number of DoFs for " +
-                                      function_space.mangled_name))
+                                      arg.name + " on " +
```

Comment:
You're right. I was trying to leave some useful information in the comment as the mangled names of `any_*_space`s have now short FS space names. I will revert it to mangled name, however, I think it would still be useful to have a full function space and argument name when we go "full symbolic" in the PSy-layer so the users can see what field is being looped over. For the future, what do you think about something like
```
" Initialise number of DoFs for <fspace._orig_name> (write arg: <arg.name>)"
```